bootstrap-salt.sh

```diff
@@ -2887,6 +2886,27 @@ install_ubuntu_check_services() {
 #
 #   Debian Install Functions
 #
+__install_saltstack_debian_repository() {
+    if [ "$DISTRO_MAJOR_VERSION" -eq 7 ]; then
+        DEBIAN_CODENAME="wheezy"
+    else
+        DEBIAN_CODENAME="jessie"
+    fi
```

Comment:
This is fine for now. But I guess in the future we would need a better checks for Debian distros codenames, something like we do for Ubuntu above and in the `__ubuntu_codename_translation` function.

~~Winter~~ Debian Stretch is coming...